src/pages/VotePage.js

```diff
@@ -0,0 +1,171 @@
+          'https://pounder-vote.shop/api/candidate/'
+        );
+        setCandidates(response.data);
+      } catch (e) {
+        console.log(1);
+      }
+    };
+    fetcthCandidates();
+  }, []);
+
+  const handleVote = (chosenCandidate) => {
+    console.log(chosenCandidate.candidate_name);
+    axios
+      .post(
+        'https://pounder-vote.shop/api/vote/',
+        { candidate: chosenCandidate.candidate_name },
+        {
+          headers: {
+            Authorization: `${localStorage.getItem('token')}`,
+            'Content-Type': 'application/json',
+          },
+        }
+      )
+      .then((response) => {
+        window.alert('투표 되었습니다!');
+        console.log(response);
+
+        setCandidates((candidates) =>
+          candidates.map((item) => {
+            if (item.candidate_name === chosenCandidate) {
+              return { ...item, vote_ctn: item.vote_ctn + 1 };
+            }
+            return item;
+          })
+        );
+      })
+
+      .catch((error) => {
+        console.log(error);
+        if (error.response.data.message === '투표는 로그인이 필요합니다!') {
+          window.alert('로그인 후 투표해주세요');
+        } else if (
+          error.response.data.message ===
+          '모든 파트의 투표를 완료하셨기에 재투표가 불가능합니다!'
+        ) {
+          window.alert('모든 파트의 투표를 완료하셨습니다!');
+        } else if (
+          error.response.data.message ===
+          '프론트엔드 파트의 투표를 완료하셨기에 재투표가 불가능합니다!'
+        ) {
+          window.alert('프론트엔드 파트의 투표를 완료하셨습니다!');
+        } else if (
+          error.response.data.message ===
+          '백엔드 파트의 투표를 완료하셨기에 재투표가 불가능합니다!'
+        ) {
+          window.alert('백엔드 파트의 투표를 완료하셨습니다!');
+        }
+      });
+  };
+
+  /*function refreshToken(){
+    const refreshtoken = localStorage.getItem('refreshtoken');
+    axios
+      .post(
+        'https://pounder-vote.shop/api/token/refresh/',
+        {
+          refresh : refreshtoken,
+        },
+        {
+          headers: {
+            'Content-type': 'application/json',
+          },
+        }
+      )
+      .then((response) => {
+        console.log(response.data);
+
+        const accesstoken = response.data.access;
+        localStorage.setItem('token', accesstoken);
+        const tokening = localStorage.getItem('token');
+        console.log(tokening)
+
+      })
+      .catch((error) => {
+        console.log(error);
+        window.alert('토큰갱신에러');
+      });
+    }*/
+
+  const logoutDB = (e) => {
+    e.preventDefault();
+    //refreshToken();
+
+    const refreshtoken = localStorage.getItem('refreshtoken');
+    console.log(token);
+    axios
+      .post(
+        'https://pounder-vote.shop/api/logout/',
+        {
+          refresh: refreshtoken,
+        },
+        {
+          headers: {
+            'Content-type': 'application/json',
+          },
+        }
+      )
+      .then((response) => {
+        console.log(response.data);
+        window.alert('로그아웃완');
+        localStorage.setItem('username', ' ');
```

Comment:
제가 이 부분을 변경했더니 로그아웃이 정상 작동하는데 한 번 해 보시겠어요?
```suggestion
        localStorage.clear();
```